### ISSUE TITLE
qclib-test man page

### ISSUE THREAD

**Issue by frank-heimes** [ibm-s390-tools/qclib#4]
Would it make sense to have a brief man page for qc_test, to face the lintian warning:
"W: qclib-test: no-manual-page [usr/bin/qc_test]" ?

**Comment by Stefan-Raspl** [comment]
Not a fan of that: It's just a test utility, and I'm not aware of distros shipping this.
Is there a way to turn the warning off?

**Comment by frank-heimes** [comment]
Yes, I could do a lintian overwrite, but I thought that I ask first (one
never knows ;-) ).

[image: Canonical-20th-anniversary]

Frank Heimes

Staff Engineer / Tech. Lead

Email:

***@***.***

Location:

Germany

canonical.com

ubuntu.com

Ubuntu on s390x Blog -- ubuntu-on-big-iron.blogspot.com
<http://ubuntu-on-big-iron.blogspot.com/?view=sidebar>
